src/themes/minimal/shape.ts

```diff
@@ -8,9 +8,9 @@ declare module '@mui/system' {
 }
 
 const shape = {
-	borderRadius: 1,
-	borderRadiusSm: 1,
-	borderRadiusMd: 2,
+	borderRadius: 8,
```

Comment:
suggesting a smaller radius for sm and md, how about 3 and 5?

Reply:
Just went with the one from figma. I'm very unopinionated about this in general - just tell me what you want to have :)